Source/SwiftyStarRatingView.swift

```diff
@@ -8,6 +8,7 @@
 
 import UIKit
 
+public typealias SSRVControlPointGenerator = (CGFloat, CGFloat) -> CGPoint
```

Comment:
Does this need to be `public`?

Reply:
Maybe not, otherwise we'll add `custom star shape path` feature.